### PR TITLE
Remove *args, **kwargs from manager public/api/_add_user_repos methods

### PR DESCRIPTION
**Pull request by gregmuellegger**
The arguments have been used wrongly at least once (see point one in #1641).

I tried to find all usages of those methods and didn't find a case were we need to catch all args/kwargs. I see that this was an attempt to unify the signatures of all the same-named methods. However IMO we should fail loudly if we an argument does not have an effect.

### CODE REVIEW

**Comment by agjohnson** [comment]
I believe the case is with our extension pattern with readthedocs.com. This PR might still be valid though, but will require an audit on the readthedocs.com side.

**Comment by ericholscher** [comment]
Yea, I agree we should reduce the scope of these methods, and they should also only care about specific privacy things, not doing any filtering otherwise.

File: readthedocs/api/base.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ class Meta(object):
     #     return bundle
 
     def get_object_list(self, request):
-        self._meta.queryset = Version.objects.api(user=request.user, only_active=False)
+        self._meta.queryset = Version.objects.api(user=request.user)
```
Comment (ericholscher) [inline readthedocs/api/base.py]: Why did you remove the only_active here? Seems like we want to include all objects, not just active.

Comment (ericholscher) [inline readthedocs/api/base.py]: Ah, I see. We're explicitly setting it, heh.

**Comment by gregmuellegger** [comment]
Ok I wasn't aware of the re-use of this code outside the codebase. Makes sense. I think the case for the PR here is minor so if it's blocking the view for more important things, feel free to close.

**Comment by ericholscher** [comment]
Gonna close this & implement it in another PR that I'm working on.
